### DISCUSSION TITLE
Version 1.86.7 released

### DISCUSSION THREAD

**Discussion by cyring** [cyring/CoreFreq#251] [General]
8664e3e0c53c1cacce389557fcc396e882668f2a is ready to be merged in `master`
See the commits log of the [`develop`](https://github.com/cyring/CoreFreq/commits/develop) branch for the changes and features.

Thanks for any report.

**Comment by cyring** [comment]
Released as `master` ad622ca5e4cdb8a503da31a1049df609cd0545be

**Comment by cyring** [comment]
@inglor 
Hi,
Not sure if you have noticed it, I have released this new version.
I hope the archive will be ok with your package.
Regards
Cyril

**Reply by inglor** [reply]
Seems perfect @cyring  Thanks for checking. Package updated.

**Reply by inglor** [reply]
As a side note I'd like to point out that the `develop` branch doesn't contain the tags from `master`. It's not really a big issue except when archlinux users are building the `corefreq-git` package which derives the version from
`<latest-tag-in-branch>.r<number-of-commits-between-latest-tag-in-branch-and-branch-head>.<commit-hash-abbreviation>`

Checkout develop and rebase from origin/master. After you should see 6 commits ahead the current branch compared with origin/develop which are the tags commits.

Something like this would work.
```
git clone https://github.com/cyring/CoreFreq.git
cd CoreFreq
git checkout -t origin/develop
git rebase origin/master
git status
...
git push origin
```

Before rebase:
```
$ git describe --long --tags                                                                                   develop
1.83-103-gad0bb47
```
After rebase:
```
$ git describe --long --tags                                                                                   develop
1.86-0-gad622ca
```

**Reply by cyring** [reply]
I will care about this. Thanks for updating packages.

**Comment by cyring** [comment]
@inglor : Hello

Version 1.86.3 released with its [tag](https://github.com/cyring/CoreFreq/releases/tag/1.86.3)

Regards,
CyrIng

**Comment by cyring** [comment]
Version 1.86.5 is available with a hotfix for Skylake/X processors

**Comment by cyring** [comment]
@inglor Hello

In addition to Skylake/X, [release 1.86.6](https://github.com/cyring/CoreFreq/releases/tag/1.86.6) is available with urgent hot-fixes for Atom[Airmont, Silvermont], Core, Core2

Regards

**Comment by cyring** [comment]
@leaty @inglor : Hello

Version [`1.86.7`](https://github.com/cyring/CoreFreq/releases/tag/1.86.7) is released for UI transparency.
